client/webserver/http.go

```diff
@@ -190,6 +196,97 @@ func (s *WebServer) handleOrders(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// handleExportOrders is the handler for the /orders/export page request.
+func (s *WebServer) handleExportOrders(w http.ResponseWriter, r *http.Request) {
+	filter := new(core.OrderFilter)
+	err := r.ParseForm()
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		return
+	}
+
+	filter.Hosts = r.Form["hosts"]
+	assets := r.Form["assets"]
+	filter.Assets = make([]uint32, len(assets))
+	for k, assetStrId := range assets {
+		assetNumId, err := strconv.Atoi(assetStrId)
+		if err != nil {
+			http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+			return
+		}
+		filter.Assets[k] = uint32(assetNumId)
+	}
+	statuses := r.Form["statuses"]
+	filter.Statuses = make([]order.OrderStatus, len(statuses))
+	for k, statusStrId := range statuses {
+		statusNumId, err := strconv.Atoi(statusStrId)
+		if err != nil {
+			http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+			return
+		}
+		filter.Statuses[k] = order.OrderStatus(statusNumId)
+	}
+
+	ords, err := s.core.Orders(filter)
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	var buffer bytes.Buffer
+	csvWriter := csv.NewWriter(&buffer)
```

Comment:
In dcrdata, starting [here](https://github.com/decred/dcrdata/blob/967628e8f638c7a673ac3f2b24195b33035d5855/explorer/explorerroutes.go#L1373) and ending [here](https://github.com/decred/dcrdata/blob/967628e8f638c7a673ac3f2b24195b33035d5855/api/apiroutes.go#L304), we are setting a special line feed for Windows clients. Should we be doing the same here?

Reply:
okay